### PR TITLE
Refactor main.py to remove excess blocks #104

### PR DESCRIPTION
**Pull request by koushikjoshi**

### CODE REVIEW

**Comment by skekre98** [comment]
@koushikjoshi really great work! Just a few pointers. Your commit tree has the changes you've made to `test.py` and `flask_tests.py`, can you please go ahead and revert these commits. 

Also since we have removed train as a parameter can you go ahead and remove instances of train in `printHelp()` such as line 11 and in line 6. Everything else looks spot on though 👍

**Comment by koushikjoshi** [comment]
> @koushikjoshi really great work! Just a few pointers. Your commit tree has the changes you've made to `test.py` and `flask_tests.py`, can you please go ahead and revert these commits.
> 
> Also since we have removed train as a parameter can you go ahead and remove instances of train in `printHelp()` such as line 11 and in line 6. Everything else looks spot on though 👍

Can you please check it now?

File: main.py [hunk 1]
```diff
@@ -3,22 +3,15 @@
 
 # Function to print help messages
 def printHelp():
-    print("usage: python main.py {test, run, data, train}")
```
Comment (skekre98) [inline main.py:6]: @koushikjoshi the entire line was not supposed to be removed. Just remove the option of `train` from the parameter list please

File: main.py [hunk 2]
```diff
@@ -3,22 +3,15 @@
...
     print("  options:")
     print("    test: runs unit tests for backend functions")
     print("    run: Runs Flask server on localhost port 5000")
     print("    data: data visualization for training sets")
-    print("    train: trains a new query classification model")
 
 def main():
 
     # Removing training history inside sqlite3 file
```
Comment (skekre98) [inline main.py:13]: Please also remove this comment as it is no longer relevant.

**Comment by skekre98** [comment]
> > @koushikjoshi really great work! Just a few pointers. Your commit tree has the changes you've made to `test.py` and `flask_tests.py`, can you please go ahead and revert these commits.
> > Also since we have removed train as a parameter can you go ahead and remove instances of train in `printHelp()` such as line 11 and in line 6. Everything else looks spot on though 👍
> 
> Can you please check it now?

@koushikjoshi One last fix and we should be good to merge 😃

**Comment by koushikjoshi** [comment]
> > > @koushikjoshi really great work! Just a few pointers. Your commit tree has the changes you've made to `test.py` and `flask_tests.py`, can you please go ahead and revert these commits.
> > > Also since we have removed train as a parameter can you go ahead and remove instances of train in `printHelp()` such as line 11 and in line 6. Everything else looks spot on though 👍
> > 
> > 
> > Can you please check it now?
> 
> @koushikjoshi One last fix and we should be good to merge 😃

Done! I'm so sorry. These were silly mistakes and I was overwhelmed 😅

**Review by skekre98 — APPROVED** [review]
LGTM!

**Pull request merged**